Gems/PhysX/Code/Source/HeightfieldCollider.h

```diff
@@ -122,6 +122,7 @@ namespace PhysX
         class HeightfieldUpdateJobContext : public AZ::JobContext
         {
         public:
+            AZ_CLASS_ALLOCATOR(HeightfieldUpdateJobContext, AZ::ThreadPoolAllocator, 0)
```

Comment:
Id the  AZ::ThreadPoolAllocator correct here?

Reply:
Yes, the JobContext base class uses [AZ::ThreadPoolAllocator](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Jobs/JobContext.h#L24)